src/renderprops/animated/Controller.js

```diff
@@ -51,6 +51,11 @@ export default class Controller {
     if (reverse) {
       ;[from, to] = [to, from]
     }
+
+    if (skipAnimation) {
+      immediate = true
```

Comment:
This isn't equivalent to the current implementation (which exists in #808):

https://github.com/react-spring/react-spring/blob/5e54f7dad6701f4179e2f16974a16a4e39840f29/packages/core/src/SpringValue.ts#L776-L778

Because it fails to avoid calling `requestAnimationFrame`.